index.html

```diff
@@ -138,11 +138,11 @@ <h3 id="form-header">Get Started</h3>
           method="POST"
           >
           <label for="name"></label>
-          <input class="st-input" type="text" name="name" placeholder="Your name" maxlength="30" required>
+          <input class="st-input" type="text" name="name" placeholder="Your name" required maxlength="30">
           <label for="email"></label>
           <input class="st-input" type="email" name="email" placeholder="Email address" required>

```

Comment:
Kindly add left padding/margin on the input areas like on the Figma template.👍
<img width="475" alt="Screenshot 2023-01-24 at 8 11 40 PM" src="https://user-images.githubusercontent.com/25887253/214364760-f7ab0dac-6ad2-4ece-a949-07630808f1fd.png">
<img width="475" alt="Screenshot 2023-01-24 at 8 12 22 PM" src="https://user-images.githubusercontent.com/25887253/214364764-e3c6cdc3-4839-4bca-b8ae-b1a7a9e6f06d.png">

Reply:
Done